### DISCUSSION TITLE
Are this series of numbers something normal in the `bun.lockb` diff?  😅

### DISCUSSION THREAD

**Discussion by nelson6e65** [oven-sh/bun#5829] [Q&A]
![bun.lockb](https://github.com/oven-sh/bun/assets/9272498/7a674316-7baf-43b0-a069-9e97a071a9b5)

The --yarn seems to be ok.

![yarn.lock](https://github.com/oven-sh/bun/assets/9272498/91e06372-e244-41a5-928f-6c609c2333d7)

**Comment by Jarred-Sumner** [comment]
Its a bug in the VSCode extension

**Reply by melroy89** [reply]
Yesn't.. bun.lockb is also a binary file. So don't expect to just open that file within an editor.
